serviceinsight/installation.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Installing ServiceInsight
+summary: Describes how ServiceInsight is installed
+component: ServiceInsight
+reviewed: 2024-06-05
+---
+
+## Prerequisites
+
+- Microsoft Windows
+- .NET Framework 4.8 or later
+- A running instance of [ServiceControl](/servicecontrol)
+
+## Installation
+
+ 1. Download and install [ServiceControl](https://github.com/Particular/ServiceControl/releases)
+ 1. Download and install [ServiceInsight](https://github.com/Particular/ServiceInsight/releases)
+ 1. After accepting the license terms and conditions, click "Install" (the installer will require elevated privileges)
+ 1. The installation process installs the "ServiceInsight" desktop application
```

Comment:
Is it worth adding a checklist on how does the user know if their installation is healthy? As in "what to expect" ?